src/bin/image-minimizer

```diff
@@ -115,37 +115,31 @@ class ImageMinimizer:
             if os.path.isdir(tag):
                 self.visited.add(tag)
             else:
-                if self.dryrun:
-                    print('rm ' + tag)
-                else:
-                    if self.verbose:
-                        print('rm ' + tag)
+                if self.dryrun or self.verbose:
+                    print("rm %s" % tag)
+                if not self.dryrun:
                     os.remove(tag)
 
         #remove all empty directory. Every 8k counts!
         for d in sorted(self.visited, reverse=True):
             if len(os.listdir(d)) == 0:
-                if self.dryrun:
-                    print('rm -rf ' + d)
-                else:
-                    if self.verbose:
-                        print('rm -rf ' + d)
+                if self.dryrun or self.verbose:
+                    print("rm -rf %s" % d)
+                if not self.verbose:
                     os.rmdir(d)
 
     def remove_rpm(self):
-
         def runCallback(reason, amount, total, key, client_data):
             if self.verbose and reason == rpm.RPMCALLBACK_UNINST_STOP:
-                print(key, "erased")
+                print("%s erased" % key)
 
         if len(self.drops_rpm) == 0:
             return
 
         for pkg in self.drops_rpm:
-            if self.dryrun:
-                print("erasing ", pkg)
-            else:
-                self.ts.addErase(pkg)
+            if self.verbose:
+                print("erasing: %s " % pkg)
+            self.ts.addErase(pkg)
```

Comment:
This will change behavior. The ``self.ts.addErase(pkg)`` is now called even in ``dryrun`` mode. Do you really want this modification?